### PR TITLE
AC4: populate dynamic list with the list of caseworkers ids for now.

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-318

### CODE REVIEW

**Review by alexbaeza — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RecordApplicationHandler.java [hunk 1]
```diff
@@ -214,7 +214,8 @@ public boolean canHandle(
                     callback.getCaseDetails().getJurisdiction(),
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RecordApplicationHandler.java:214]: Do we have unit test coverage for this?

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/PreSubmitCallbackDispatcher.java [hunk 2]
```diff
@@ -81,7 +81,8 @@ public PreSubmitCallbackDispatcher(
                         callback.getCaseDetails().getJurisdiction(),
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/PreSubmitCallbackDispatcher.java:81]: Do we have unit test coverage for this?

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/RoleAssignmentApi.java [hunk 3]
```diff
@@ -7,7 +7,9 @@
 import org.springframework.web.bind.annotation.PostMapping;
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/RoleAssignmentApi.java:7]: Do we have unit test coverage for the changes introduced?

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RecordApplicationHandler.java (hunk 1 above)
Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RecordApplicationHandler.java:214]: added here CaseDetailsTest

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/PreSubmitCallbackDispatcher.java (hunk 2 above)
Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/PreSubmitCallbackDispatcher.java:81]: added here CaseDetailsTest

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/RoleAssignmentApi.java (hunk 3 above)
Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/RoleAssignmentApi.java:7]: We don't cover other Api clients directly but as part of the junit test for the class that uses the API. This one, in particular, is covered by this test RoleAssignmentServiceTest

**Pull request merged**